MAINTAINERS.yml

```diff
@@ -1410,6 +1410,22 @@ Espressif Platforms:
     labels:
         - "platform: ESP32"
 
+ITE Platforms:
+    status: orphaned
+    collaborators:
+        - RuibinChang
+        - DinoLi
+        - GTLin08
+        - jackrosenthal
+        - Keith Short
+        - brockus-zephyr
+        - Yuval Peress
+        - sjg20
+    files:
+        - dts/riscv/
```

Comment:
need to be more specific about the files here here, this covers all riscv but non of the ITE related platforms, SoCs.

Reply:
Done